src/main/java/apaintus/models/ListViewCell.java

```diff
@@ -0,0 +1,48 @@
+package apaintus.models;
```

Comment:
Should be in apaintus/views and not models

Reply:
So we re creating a new folder for this, correct ?